src/modules/leavePage.js

```diff
@@ -0,0 +1,28 @@
+import { classify } from '../helpers.js';
+
+const leavePage = function(data, { popstate = false, skipTransition = false } = {}) {
+	if (skipTransition) {
+		this.triggerEvent('animationSkipped');
+		return [Promise.resolve()];
+	}
+
+	this.triggerEvent('animationOutStart');
+
+	// handle classes
+	document.documentElement.classList.add('is-changing');
+	document.documentElement.classList.add('is-leaving');
+	document.documentElement.classList.add('is-animating');
+	if (popstate) {
+		document.documentElement.classList.add('is-popstate');
+	}
+	document.documentElement.classList.add(`to-${classify(data.url)}`);
```

Comment:
Maybe this could have some util function as well, similar to the `cleanupAnimationClasses` introduced? I'm thinking something like, but maybe there is better options too. 

```javascript
addAnimationClasses('is-changing', 'is-leaving', 'is-animating', `to-${classify(data.url)}`);
if (popstate) addAnimationClasses('is-popstate');
```

Reply:
Good idea! The repetition of `document.documentElement` is a good pointer here for sure.